yarn.lock

```diff
@@ -2961,9 +2961,9 @@
     "@openzeppelin/contracts" "^4.8.0"
     "@paulrberg/contracts" "^3.7.0"
 
-"@jbx-protocol/juice-ownable@git+https://github.com/jbx-protocol/juice-ownable.git":
```

Comment:
im unsure why I keep getting this file

Reply:
Can we just remove it from the commit do you think? And have you `yarn pristine`d?